spec/system/service_unavailable_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Service Unavailable' do
+  let(:user) { create :user }
```

Comment:
Where is `user` in play?

Reply:
[796f7c7](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/941/commits/796f7c7bfd6504680528e11e99c3cbeaee5a27d0)